### PR TITLE
Gogoanime presence (.io and .video)

### PR DESCRIPTION
**Pull request by PsychoPast**
image

### CODE REVIEW

File: Gogoanime/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "en": "Watch anime online in English. You can watch free series and movies online and English subtitle."
+    },
+  "url": [
+    "www2.gogoanime.video",
+    "www17.gogoanime.io"
+    
+  ],
+  "version": "1.0.0",
+  "logo": "https://cdn.psychopast.tk/files/gogoanime-logo.png?key=zsBPy7m=t79yBOvQQF5gKZRoRcpoi+kn",
+  "thumbnail": "https://cdn.psychopast.tk/files/gogoanime-thumb.png?key=zsBPy7m=t79yBOvQQF5gKZRoRcpoi+kn",
```
Comment (Alanexei) [inline Gogoanime/dist/metadata.json:17]: Do you own psychopsast.tk?

Comment (PsychoPast) [reply Gogoanime/dist/metadata.json:17]: Yes :)

File: Gogoanime/dist/metadata.json [hunk 2]
```diff
@@ -13,8 +13,8 @@
     
   ],
   "version": "1.0.0",
-  "logo": "https://cdn.psychopast.tk/files/gogoanime-logo.png?key=zsBPy7m=t79yBOvQQF5gKZRoRcpoi+kn",
-  "thumbnail": "https://cdn.psychopast.tk/files/gogoanime-thumb.png?key=zsBPy7m=t79yBOvQQF5gKZRoRcpoi+kn",
+  "logo": "https://i.imgur.com/Awp1MUc.png",
+  "thumbnail": "https://i.imgur.com/MmCT2wf.png",
```
Comment (frozaum) [inline Gogoanime/dist/metadata.json:17]: can you use a promotional image or a screenshot of the website instead?

Comment (PsychoPast) [reply Gogoanime/dist/metadata.json]: sure :)

File: Gogoanime/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "service": "Gogoanime",
+  "description": {
+    "en": "Watch anime online in English. You can watch free series and movies online and English subtitle."
+    },
+  "url": [
+    "www2.gogoanime.video",
+    "www17.gogoanime.io"
+    
+  ],
```
Comment (Bas950) [inline Gogoanime/dist/metadata.json:14]: this enter triggers me

**Pull request merged**